jobs/gtfs-rt-parser-v2/gtfs_rt_parser.py

```diff
@@ -309,20 +310,19 @@ def download_gtfs_schedule_zip(
     pbar=None,
 ) -> str:
     # fetch and zip gtfs schedule
-    actual_dst_path = "/".join([dst_path, schedule_extract.filename])
     log(
-        f"Fetching gtfs schedule data from {schedule_extract.path} to {actual_dst_path}",
+        f"Fetching gtfs schedule data from {schedule_extract.path} to {dst_path}",
         pbar=pbar,
     )
-    get_with_retry(fs, schedule_extract.path, actual_dst_path, recursive=True)
+    get_with_retry(fs, schedule_extract.path, dst_path, recursive=True)
 
     # https://github.com/MobilityData/gtfs-realtime-validator/issues/92
     # try:
     #     os.remove(os.path.join(dst_path, "areas.txt"))
     # except FileNotFoundError:
     #     pass
 
-    return actual_dst_path
+    return "/".join([dst_path, schedule_extract.filename])
```

Comment:
is this the bug mentioned in PR description? just curious what it was

Reply:
oh yeah will mention